### ISSUE TITLE
Training invalidation when add the nubmer of layers

### ISSUE THREAD

**Issue by staplesinLA** [Microsoft/CNTK#476]
Hello, i trained a DNN with 6 hidden layers * 2048 nodes and it worked well. Its activation function is Sigmoid.
But when i added  a hidden layer( 6 layers to 7 layers ) , the training stucked at the begining.The training error don't descend at all. I know too deep layers may occur this issue, but training result seems  much worse than i thought. Beacause it's good with 6 layers, and i used Sigmoid because i wanted to reproduce a experiment result by others.
Is this a normal situation?
And, the pretraining sample given by CNTKBook is actually kindof a supervised way?

**Comment by dongyu888** [comment]
Use a smaller learning rate for the first epoch and it usually will solve your problem.

**Comment by staplesinLA** [comment]
@dongyu888 thank you very much. Finally i choose to do some epochs for discriminative pretraining and it becomes more stable.
